scripts/dist_build.sh

```diff
@@ -16,10 +16,15 @@ GIT_IMPORT="github.com/tendermint/tendermint/version"
 # Determine the arch/os combos we're building for
 XC_ARCH=${XC_ARCH:-"386 amd64 arm"}
 XC_OS=${XC_OS:-"solaris darwin freebsd linux windows"}
+XC_EXCLUDE=${XC_EXCLUDE:-" darwin/arm solaris/amd64 solaris/386 solaris/arm freebsd/amd64 windows/arm "}
 
 # Make sure build tools are available.
-# TODO: Tools should be "vendored" too.
-make get_tools
+curl -L https://github.com/Masterminds/glide/releases/download/v0.13.1/glide-v0.13.1-linux-amd64.tar.gz | tar xvfz - && \
```

Comment:
lets be paranoid and compare sha256sum here

Reply:
we need to call make get_tools here.  Otherwise we have two differing behaviors for downloading tools.

Reply:
`go get -u github.com/tendermint/glide` should be sufficient.

Reply:
it's not completely deterministic, but ok